### PR TITLE
feat(scrollviewer): Add support of touch in managed ScrollViewer

### LINKED ISSUE #4936: Touch support for Skia GTK backend
## What would you like to be added:

- [ ] Touch support for the GTK head
- [ ] ScrollViewer support for Scrolling

### PR DESCRIPTION
**Pull request by dr1rrb**
Fixes #4936
Complete work started with https://github.com/unoplatform/uno/pull/5429

## Feature
Add ability to scroll with finger on skia

## What is the current behavior?
`ScrollViewer` was not handling the touch manipulations

## What is the new behavior?
🙃

## PR Checklist
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] ~~[Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)~~
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs [hunk 1]
```diff
@@ -224,7 +235,11 @@ private void ScrollContentPresenter_PointerWheelChanged(object sender, Input.Poi
 				var canScrollHorizontally = HorizontalScrollBarVisibility != ScrollBarVisibility.Disabled;
 				var canScrollVertically = VerticalScrollBarVisibility != ScrollBarVisibility.Disabled;
 
-				if (!canScrollVertically || properties.IsHorizontalMouseWheel || e.KeyModifiers.HasFlag(global::Windows.System.VirtualKeyModifiers.Shift))
+				if (e.KeyModifiers == VirtualKeyModifiers.Control)
+				{
+					// TODO: Handle zoom
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs:240]:
```suggestion
					// TODO: Handle zoom https://github.com/unoplatform/uno/issues/4309
```

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseScrollContentPresenter.cs [hunk 2]
```diff
@@ -60,6 +60,18 @@ public ScrollBarVisibility VerticalScrollBarVisibility
 			}
 		}
 
+		public bool CanHorizontallyScroll
+		{
+			get => HorizontalScrollBarVisibility != ScrollBarVisibility.Disabled;
+			set { }
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseScrollContentPresenter.cs:66]: ?

**Review by davidjohnoliver — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseScrollContentPresenter.cs (hunk 2 above)
Comment (dr1rrb) [reply src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseScrollContentPresenter.cs:66]: @davidjohnoliver  This property is part of the public contract on the SCP (including the `set` :( ), while the `HorizontalScrollBarVisibility` is not. As I was needing this, I've update the ISCP interface to better reflect the SCP contract, but didn't changed the implementation and only re-route to the old property.

**Pull request merged**